docs/src/architecture/accounts.md

```diff
@@ -80,3 +76,10 @@ The process is as follows:
 * Bob executes a transaction and creates a note that contains an asset for Alice
 * Alice consumes Bob's note to receive the asset in a transaction
 * Depending on the account storage mode (private vs. public) and transaction type (local vs. network) the Operator receives new Account ID eventually and - if transaction is correct - adds the ID to the Account DB
+
+## Account storage modes
+Account data - stored by the Miden Node - can be public, private, or encrypted. The third and fourth most significant bits of the account ID specifies whether the account data is public `00`, encrypted `01`, or private `11`.
+
+* Accounts with **public state**, where the actual state is stored onchain. These would be similar to how accounts work in public blockchains. Smart contracts that depend on public shared state should be stored public on Miden, e.g., DEX contract.
+* Account with **encrypted state**, where the account data is stored onchain but in encrypted text. When there are multiple owners of an account that still want to use their account privately together, they should choose this mode.  
```

Comment:
I think encrypted accounts are useful beyond the multiple owner use case.  It provides liveness guarantee of the protocol for the account in question.

Reply:
Very well. This needs to change in a separate PR. This PR is only about notes, it just didn't rebase properly. Will change in a separate PR.